src/App.js

```diff
@@ -1,156 +1,188 @@
             </div>
-            <div className="slider__button">
-              <div className="slider__button-caption">02</div>
-              <div className="slider__button-icon"></div>
-            </div>
-            <div className="slider__button">
-              <div className="slider__button-caption">03</div>
-              <div className="slider__button-icon"></div>
-            </div>
-          </div>
-        </div>
-      </div>
 
-      <div className="search">
-        <div className="search__icon">
-          <img src="/plane.png" alt="icon" />
-        </div>
-        <div className="search__filters">
-          <div className="filter">
-            <div className="filter__caption">Departure airport</div>
-            <div className="filter__value">Mumbai (BOM) &darr;</div>
-          </div>
-          <div className="filter">
-            <div className="filter__caption">Arrival airport</div>
-            <div className="filter__value">Aalborg (AAL) &darr;</div>
           </div>
         </div>
-        <button className="search__btn">Search</button>
       </div>
 
-      <div className="pages">
-        <div className="pages__transition"></div>
-        <div className="pages__content">
-          <div className="column-section">
-            <div className="column-section__column">
-              <h1 className="heading--secondary-branded">The Zabdos Experience</h1>
-              <div className="column-section__description">
-                Wheter it's your first flight or simply latest, we work to anticipate your every
-                need
-              </div>
-              <div className="column-section__links">
-                <a href="c" className="link-with-icon">
-                  Watch the full video now!
-                </a>
-              </div>
-            </div>
-            <div className="column-section__column">
-              <div className="card">
-                <img src="/inside1.jpg" alt="inside" />
-                <div className="card__subcaption">Arrive inspired</div>
-                <div className="card__caption">Business Class</div>
-              </div>
-              <div className="card">
-                <img src="/inside2.jpg" alt="inside" />
-                <div className="card__subcaption">Expect the exceptional</div>
-                <div className="card__caption">Economy Class</div>
-              </div>
-            </div>
-            <div className="column-section__column">
-              <div className="card--visible-on-top">
-                <img src="/vr.jpg" alt="vr" />
-                <div className="card__subcaption">Award winning technology</div>
-                <div className="card__caption">Explore the Fleet in VR</div>
-              </div>
-            </div>
-          </div>
+      <div className="secondary-content">
+        <div className="centered-container">
+          <div className="main-container">
+            <div className="indented-container">
```

Comment:
Way too many wrappers, or the so-called `div soup` :) As mentioned before you can get rid of the `main-container`, plus the `secondary-content` can go as well